servers/store/src/main/java/org/projectnessie/server/store/TableCommitMetaStoreWorker.java

```diff
@@ -294,6 +334,29 @@ public boolean requiresGlobalState(ByteString content) {
     }
   }
 
+  @Override
+  public boolean requiresPerContentState(Content content) {
+    if (content instanceof IcebergTable) {
+      IcebergTable table = (IcebergTable) content;
+      return table.getMetadata() != null;
+    }
+    if (content instanceof IcebergView) {
+      IcebergView view = (IcebergView) content;
+      return view.getMetadata() != null;
+    }
+    return false;
+  }
+
+  @Override
+  public boolean requiresPerContentState(ByteString content) {
+    return false;
+  }
+
+  @Override
+  public boolean requiresPerContentState(Enum<Content.Type> type) {
+    return type == Content.Type.ICEBERG_TABLE || type == Content.Type.ICEBERG_VIEW;
```

Comment:
What is the difference wrt `requiresPerContentState(Content content)`? There are no callers, so it is hard to tell :) 

The other method is specific to the content value, but this one is based on type only... Would it make sense to rename it e.g. to `canHavePerContentState` (or to `canHaveContentAttachments`)?

Reply:
Removed all `requiresPerContentState` functions - those are no longer needed